yourgov/components/FormDefinitionList.tsx

```diff
@@ -85,34 +88,40 @@ export function FormDefinitionListItemDate({
 }
 
 type FormDefinitionListItemDateRangeProps = {
-	label: string;
-	from: unknown | undefined;
-	to: unknown | undefined;
+	fromLabel: string;
+	fromValue?: unknown;
+	toLabel: string;
+	toValue?: unknown;
```

Comment:
Hmm these two `unknown`s aren't right.

Reply:
Looking at the schema they should only be `Date`s or `undefined`.

Reply:
Oh, the `DeepPartial` util is too aggressive. It's modifying the native `Date` type. We need to modify /yourgov/lib/types.ts - this will be safer as it won't modify the `Date`s:

```ts
export type DeepPartial<T> = T extends Date
	? T
	: T extends object
	? {
			[P in keyof T]?: DeepPartial<T[P]>;
	  }
	: T;
```

Reply:
Then these lines can be:

```suggestion
	fromValue?: Date;
	toLabel: string;
	toValue?: Date;
```